Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ui/ViewExtensions.kt

```diff
@@ -4,6 +4,11 @@ import android.view.View
 import androidx.databinding.BindingAdapter
 import de.rki.coronawarnapp.util.recyclerview.ThrottledClickListener
 
+@BindingAdapter("visible")
+fun View.setVisible(visible: Boolean) {
```

Comment:
This is exactly  the same but negated as the `setGone` below

Reply:
See other comment.